runtime/exports.t4

```diff
@@ -177,23 +177,23 @@
 
 		#region metadata/profiler.h
 
-		new Export ("void", "mono_profiler_set_events",
-			"MonoProfileFlags", "events"
+		new Export ("MonoProfilerHandle", "mono_profiler_install",
```

Comment:
This hopefully won't happen again, at all. This was an extremely rare case of us breaking compatibility in Mono's embedding API because the old API was fundamentally broken.